lib/spec/appliance_console/service_group_spec.rb

```diff
@@ -5,78 +5,149 @@
 require "linux_admin"
 
 describe ApplianceConsole::ServiceGroup do
-  before do
-    @group = described_class.new
-    @common_services  = %w{evminit memcached miqtop evmserverd}
-    @postgres_service = %w{postgresql92-postgresql}
-  end
+  let(:group)  { described_class.new }
+  let(:common_services)   { %w(evminit memcached miqtop evmserverd) }
+  let(:postgres_service)  { "postgresql92-postgresql" }
+
+  describe "#postgresql?" do
+    it { expect(group).not_to be_postgresql }
 
-  it "#postgresql?" do
-    expect(described_class.new(:internal_postgresql => true).postgresql?).to be_true
-    expect(described_class.new(:internal_postgresql => false).postgresql?).to be_false
-    expect(described_class.new.postgresql?).to be_false
+    context "when internal postgres" do
+      let(:group) { described_class.new(:internal_postgresql => true) }
+      it { expect(group).to be_postgresql }
+    end
+
+    context "when not internal postgres" do
+      let(:group) { described_class.new(:internal_postgresql => false) }
+      it { expect(group).not_to be_postgresql }
+    end
   end
 
-  context "postgresql" do
-    before do
-      @group.stub(:postgresql? => true)
+  describe "#enable" do
+    it "enables all but postgres" do
+      expect(group).to receive(:enable_miqtop)
+      common_services.each do |service|
+        expect_run_service(service, "enable")
```

Comment:
Does setting the expectations of just these services cause a failure if postgresql is also "enabled"?  In other words, we don't explicitly say that postgresql wont' be called here.